drivers/soundwire/intel.c

```diff
@@ -401,6 +401,45 @@ static void intel_shim_wake(struct sdw_intel *sdw, bool wake_enable)
 	}
 }
 
+void sdw_intel_process_wakeen_event(struct sdw_intel_ctx *ctx)
```

Comment:
typo in commit message "tigger a"->"trigger an"

Reply:
thanks! I will fix it.